orx-depth-camera/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+    kotlin("multiplatform")
+}
+
+kotlin {
+    jvm {
+        compilations.all {
+            kotlinOptions.jvmTarget = libs.versions.jvmTarget.get()
+            kotlinOptions.apiVersion = libs.versions.kotlinApi.get()
+        }
```

Comment:
Conflicts with #256. If #256 is merged first, the `compilations.all` block should be removed in this PR.

Reply:
@edwinRNDR mentioned that 0.4.1 should go first. So probably I will update this PR once it is released.

Reply:
@Vechro done